build-on-push/platform_ci/platform_ci/distgit_test.py

```diff
@@ -17,20 +17,22 @@
 # pylint: disable=no-name-in-module
 from nose.tools import assert_raises
 
-from .distgit import DistGitBranch, DistGitBranchException
+from platform_ci.distgit import DistGitBranch, DistGitBranchException
 
 
 class DistGitBranchTest(unittest.TestCase):
-    STAGING = {"branch": "rhel-7.1-staging", "target": "rhel-7.1-candidate"}
+    STAGING = {"branch": "staging-rhel-7", "target": "staging-rhel-7-candidate"}
     PRIVATE = {"branch": "private-pmuller-branch", "target": None}
-    STANDARD = {"branch": "extras-rhel-7.2", "target": "extras-rhel-7.2-candidate"}
-    EXTRAS_STAGING = {"branch": "extras-rhel-7.2-staging", "target": "extras-rhel-7.2-candidate"}
+    STANDARD = {"branch": "rhel-7.2", "target": "rhel-7.2-candidate"}
+    EXTRAS_STANDARD = {"branch": "extras-rhel-7.2", "target": "extras-rhel-7.2-candidate"}
```

Comment:
As long as the underlying branch is tied to a minor-version of a product (like all standard branches: we have "rhel-7.2" branch, "extras-rhel-7.2" branch etc) they should be build against a matching target: it does not make sense to build  "rhel-7.2" branch against "rhel-7-candidate" because that one is likely to point to a tag inheriting rhel-7.3-candidate.

Currently, BoP does not auto-build standard branches anyway, so it does not matter much.